cloudevents/tests/test_http_cloudevent.py

```diff
@@ -16,48 +21,46 @@ def test_http_cloudevent_equality(specversion):
         "datacontenttype": "application/json",
         "subject": "my-subject",
     }
-    data = '{"name":"john"}'
-    event1 = CloudEvent(attributes, data)
-    event2 = CloudEvent(attributes, data)
+
+
+_my_dummy_data = '{"name":"john"}'
+_your_dummy_data = '{"name":"paul"}'
+assert _my_dummy_data != _your_dummy_data
```

Comment:
Can you please avoid having any runnable things and assertions in the file scope?

I'd highly recommend migrating these tests to Class-like test cases.